recipe-server/bin/create_test_admin.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
Django has no way to create a superuser in a script _with a password_. `createsuperuser` can only set a password interactively.

The common alternative is to pipe the Python code to `manage.py shell`, but I had lots of trouble getting this to work in a container run via `docker-compose`. I'd love to do this better.